data-charting.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: default
+title: Data charting
+nav_order: 3
+parent: API
+---
+
+## Data charting
+
+For context, please read [Calling the API page](/calling-the-api) before going through this page.
+
+Assuming you have already obtained the API credentials and were able to receive an API response with some data for a specific location, you may now also want to create charts with the percentiles that represent the full range of expected weather outcomes for that location in various warming scenarios. As described on the [Calling the API page](/calling-the-api), the variable `x` that you receive from the API contains 101 variables corresponding to the precentiles ranging from 0 to 100. Below you will find more information on how to use this variable to draw plots.
+
+Let start with an example:
+
+- Call the API to retreive data about CITY, Arizona.
```

Comment:
@MoustafaWehbe, I think we should use a specific city in this example because Arizona is big and has lots of different weather in different places. How about Phoenix, Arizona? If that sounds good to you, could you please update the language (where it says CITY), x values in the response, and screenshot to reflect that?

Reply:
sure